ci/deploy/deploy-pipeline.yaml

```diff
@@ -197,4 +197,138 @@ spec:
                     --allow-ns "${RELEASE_NAMESPACE}" \
                     --app "${RELEASE_NAME}-${APP_NAME}" \
                     --diff-changes \
-                    -f ./manifests/
\ No newline at end of file
+                    -f ./manifests/
+    
+    - name: deploy-production
+        serial: true
+        plan:
+
+          - get: release
+            trigger: true
+
+          - get: daily
+            trigger: true
+
+          - get: eidas-config-production
```

Comment:
The resource `eidas-config-production` doesn't exist. Think we should rename the resource to `verify-eidas-config` and update both the integration and prod releases to use this new name